src/Test/Perf/helloworld/HelloWorld.csx

```diff
@@ -0,0 +1,20 @@
+#load "../util.csx"
+using System.IO;
+
+var pathToCsc = Path.Combine(BinDebugDirectory(), "csc.exe");
```

Comment:
Consider extracting this as a global property that all tests can use.  It would scare me a little to have different tests testing compilers from diff locations.